atomic_reactor/plugin.py

```diff
@@ -210,6 +214,9 @@ def __init__(self, dt, workflow, plugin_class_name, plugins_conf, *args, **kwarg
         self.workflow = workflow
         super(BuildPluginsRunner, self).__init__(plugin_class_name, plugins_conf, *args, **kwargs)
 
+    def on_plugin_failed(self):
+        self.workflow.build_has_failed = True
```

Comment:
I'm sorry I still don't follow. I thought that setting `build_has_failed` makes only sense in `inner.py` where we figure out if the build successful.